iota/commands/extended/get_new_addresses.py

```diff
@@ -36,25 +36,35 @@ def _execute(self, request):
     count           = request['count'] # type: Optional[int]
     index           = request['index'] # type: int
     security_level  = request['securityLevel'] # type: int
+    checksum        = request['checksum'] # type: Optional[bool]
     seed            = request['seed'] # type: Seed
 
     return {
-      'addresses': self._find_addresses(seed, index, count, security_level),
+      'addresses': 
+         self._find_addresses(seed, index, count, security_level, checksum),
     }
 
-  def _find_addresses(self, seed, index, count, security_level):
-    # type: (Seed, int, Optional[int], int) -> List[Address]
+  def _find_addresses(self, seed, index, count, security_level, checksum):
+    # type: (Seed, int, Optional[int], int, Optional[bool]) -> List[Address]
     """
     Find addresses matching the command parameters.
     """
-    # type: (Seed, int, Optional[int]) -> List[Address]
-    generator = AddressGenerator(seed, security_level)
+    # type: (Seed, int, Optional[bool]) -> List[Address]
```

Comment:
I think this guy's a typo, actually.  The type hint goes above the docstring, not under it ([PEP-484](https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code)).  You can remove this line entirely.